File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int n= s.length();
        int j= n-1 , ans=0;
        while(j>=0 &&s[j]==' ')
        {
            j--;
        }
    
        while(j>=0&& s[j]!=' ')
        {
            j--;
            ans++;
        }
        return ans;
        
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int n= s.length();
        
        int j=n-1, ans=0;
        while(j>=0 && s.charAt(j)==' ')
        {
            j--;
        }
        while(j>=0 && s.charAt(j)!= ' ')
        {
            j--; 
            ans++;
        }
        return ans;
    }
}